### PR TITLE
Fix identity authentication

### PR DESCRIPTION
**Pull request by askolesov**

### CODE REVIEW

File: x/cheqd/types/keys.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ func KeyPrefix(p string) []byte {
 const (
 	DidKey      = "did:"
 	DidCountKey = "did-count:"
+	// DidNamespaceKey FIXME: Should be `did-namespace:`.
+	// Networks was started with `testnet` value so we need a migration now.
+	DidNamespaceKey = "testnet"
```
Comment (ankurdotb) [inline x/cheqd/types/keys.go:30]: This looks like a regression?

**Review by ankurdotb — COMMENTED** [review]
Will leave this to @Toktar and @anikitinDSR to look in detail

File: x/cheqd/types/keys.go (hunk 1 above)
Comment (askolesov) [reply x/cheqd/types/keys.go:30]: No, a wrong constant was used to take this value, I fixed that and made the fact that we used the wrong value more explicit

**Review by Toktar — APPROVED** [review]

File: tests/cli/tests/test_rotate_key.sh [hunk 2]
```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+set -euox pipefail
+
+# sed in macos requires extra argument
+
+sed_extension=''
```
Comment (anikitinDSR) [inline tests/cli/tests/test_rotate_key.sh:7]: Is it still needed?

**Pull request merged**